src/app/App.svelte

```diff
@@ -1,13 +1,14 @@
 <script lang="ts">
-  export let name: string;
-
+  // export let name: string;
+  import Languages from "./Languages.svelte";
 </script>
 
 <main role="application">
-  <h1> Welcome to {name}!</h1>
+  <!-- <h1> Welcome to {name}!</h1>
```

Comment:
```suggestion
```

Reply:
GitHub wouldn't let me select the code inbetween this and the `-->`, but basically, delete that too!